stories/cosmoz-input.stories.js

```diff
@@ -46,16 +46,24 @@ export const contour = () => html`
 	<style>
 		cosmoz-input {
 			--cosmoz-input-border-radius: 4px;
-			--cosmoz-input-padding: 8px;
+			--cosmoz-input-padding: 10px;
 			--cosmoz-input-label-width: auto;
-			--cosmoz-input-no-placeholder-label-bg: white;
 			--cosmoz-input-label-padding: 0;
 			--cosmoz-input-line-display: none;
 			--cosmoz-input-contour-size: 1px;
-			--cosmoz-input-background: white;
+			--cosmoz-input-label-translate-y: 35%;
```

Comment:
```suggestion
			--cosmoz-input-label-translate-y: 43%;
```

I think that moving things a bit lower leads to a better result:
[vokoscreenNG-2024-03-08_11-36-24.webm](https://github.com/Neovici/cosmoz-input/assets/710155/d1b9d9d5-a466-4dce-8a5a-a6f2dd98d6b5)